cfgov/unprocessed/js/modules/transition/BaseTransition.js

```diff
@@ -198,9 +203,9 @@ function BaseTransition( element, classes ) { // eslint-disable-line max-stateme
   this.setElement = setElement;
 
   // Public static constants.
-  BaseTransition.BEGIN_EVENT = 'transitionBegin';
-  BaseTransition.END_EVENT = 'transitionEnd';
-  BaseTransition.NO_ANIMATION_CLASS = 'u-no-animation';
+  this.BEGIN_EVENT = BEGIN_EVENT;
+  this.END_EVENT = END_EVENT;
+  this.NO_ANIMATION_CLASS = NO_ANIMATION_CLASS;
```

Comment:
I know it's IE :rage4:, but this doesn't look like it's a necessary change. It looks like these can just be moved outside the function definition, so that the end of the file would look like:

```

  return this;
}

// Public static constants.
BaseTransition.BEGIN_EVENT = 'transitionBegin';
BaseTransition.END_EVENT = 'transitionEnd';
BaseTransition.NO_ANIMATION_CLASS = 'u-no-animation';

module.exports = BaseTransition;
```

If you then put this at the top of common.js to test, it works in IE9:

```

var BaseTransition = require( '../modules/transition/BaseTransition' );
console.log( BaseTransition.BEGIN_EVENT );
```